dockstore-client/pom.xml

```diff
@@ -529,6 +529,8 @@
                         --add-opens java.base/java.io=ALL-UNNAMED
                         --add-opens java.base/java.util=ALL-UNNAMED
                         --add-opens java.base/java.base=ALL-UNNAMED
+                        <!-- for  Java 21, see   -->
```

Comment:
Incomplete comment?

Reply:
extended comment, needed for system stubs in Java 21